frontend/src/scenes/urls.ts

```diff
@@ -7,12 +7,20 @@ export const urls = {
     notFound: () => '404',
     dashboards: () => '/dashboard',
     dashboard: (id: string | number) => `/dashboard/${id}`,
-    createAction: () => `/action`, // TODO: For consistency, this should be `/action/new`
-    action: (id: string | number) => `/action/${id}`,
-    actions: () => '/events/actions',
-    eventStats: () => '/events/stats',
-    eventPropertyStats: () => '/events/properties',
-    events: () => '/events',
+    // TODO: Deprecate below with `collaborations-taxonomy` FF
+    LEGACY_createAction: () => `/action`, // TODO: For consistency, this should be `/action/new`
+    LEGACY_action: (id: string | number) => `/action/${id}`,
+    LEGACY_actions: () => '/events/actions',
+    LEGACY_eventStats: () => '/events/stats',
+    LEGACY_eventPropertyStats: () => '/events/properties',
+    LEGACY_events: () => '/events',
+    // TODO: Deprecate above with `collaborations-taxonomy` FF
+    events: () => '/events_v2', // live events
```

Comment:
Why not something like `live_events` to be consistent with the sidebar and page name.